webapp/models/location.py

```diff
@@ -108,8 +108,7 @@ class Location(db.Model, BaseModel):
         Index('uid', 'source'),
     )
 
-    uid: Mapped[str] = db.Column(db.String(255), index=True,
-                              nullable=False)  # OCHP: locationId                      OCPI: id
+    uid: Mapped[str] = db.Column(db.String(255), index=True,nullable=False)  # OCHP: locationId   OCPI: id
```

Comment:
PEP 8: E231 missing whitespace after ',' (betweeen index and nullable)

Reply:
fixed :)